Makefile

```diff
@@ -459,17 +454,17 @@ deploy-snr:
 
 ##@ Targets used by CI
 
-.PHONY: container-build
-container-build: ## Build containers
+.PHONY: container-build-ocp
+container-build-ocp: ## Build containers
```

Comment:
NIT: Can you modify the comment after the target to be more specific (and below)

```suggestion
container-build-ocp: ## Build containers for OCP Prometheus
```